docs/resources/connection.md

```diff
@@ -111,8 +117,10 @@ resource "steampipecloud_connection" "aws_role_connection" {
 resource "steampipecloud_connection" "gcp_aaa" {
   handle      = "gcp_aaa"
   plugin      = "gcp"
-  project     = "project-aaa"
-  credentials = file("/Users/myuser/Downloads/project-aaa.json")
+  config = jsonencode({
+    project     = "project-aaa"
+    credentials = file("/Users/myuser/Downloads/project-aaa.json")
```

Comment:
```suggestion
    credentials = jsondecode(file("/Users/myuser/Downloads/project-aaa.json"))
```

I know we have special handling in today for GCP and OCI creds, but after talking with @johnsmyth a bit, we were wondering does the connection get created correctly if we try to decode the file content first?